angular/test/apps/ng15/package.json

```diff
@@ -20,19 +20,18 @@
     "test.watch": "concurrently \"npm run start\" \"wait-on http-get://localhost:4200 && npm run cy.open\" --kill-others --success first"
   },
   "dependencies": {
-    "@angular/animations": "^15.0.0-rc.1",
-    "@angular/common": "^15.0.0-rc.1",
-    "@angular/compiler": "^15.0.0-rc.1",
-    "@angular/core": "^15.0.0-rc.1",
-    "@angular/forms": "^15.0.0-rc.1",
-    "@angular/platform-browser": "^15.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "^15.0.0-rc.1",
-    "@angular/platform-server": "^15.0.0-rc.1",
-    "@angular/router": "^15.0.0-rc.1",
+    "@angular/animations": "^15.0.0",
+    "@angular/common": "^15.0.0",
+    "@angular/compiler": "^15.0.0",
+    "@angular/core": "^15.0.0",
+    "@angular/forms": "^15.0.0",
+    "@angular/platform-browser": "^15.0.0",
+    "@angular/platform-browser-dynamic": "^15.0.0",
+    "@angular/platform-server": "^15.0.0",
+    "@angular/router": "^15.0.0",
     "@ionic/angular": "^6.1.15",
     "@ionic/angular-server": "^6.1.15",
-    "@nguniversal/express-engine": "^14.0.3",
-    "angular-in-memory-web-api": "^0.11.0",
```

Comment:
This dependency was unused and was causing problems. I'll have a follow-up PR to remove it from the other test apps (it is unused in all).